config/redis.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const config = require('config');
+
+const redisConfig = config.get('redis');
+
+const queuePrefix = redisConfig.prefix || '';
+
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: redisConfig.host,
+    options: {
+        password: redisConfig.password
+    },
+    port: redisConfig.port,
+    database: 0
+};
+
+module.exports = {
```

Comment:
Specifically, this test should be different: https://github.com/screwdriver-cd/queue-worker/blob/d9f0dd2f42c394bb43b3cb63955451b50d9ed115/test/index.test.js#L196